lib/rubocop/cop/factory_bot/consistent_parentheses_style.rb

```diff
@@ -79,7 +79,7 @@ class ConsistentParenthesesStyle < ::RuboCop::Cop::Base
         def_node_matcher :omit_hash_value?, <<~PATTERN
           (send
             #factory_call? %FACTORY_CALLS
-            {sym str send lvar}
+            {sym str send lvar} _*
```

Comment:
First time working with AST. So there may be some implications here that I am missing. The intention is to allow any number of traits to be used before the hash values in the calls.